images/tomcat/configs/latest.apko.yaml

```diff
@@ -0,0 +1,69 @@
+contents:
+  keyring:
+    - https://packages.wolfi.dev/os/wolfi-signing.rsa.pub
+  repositories:
+    - https://packages.wolfi.dev/os
+  packages:
+    - wolfi-baselayout
+    - ca-certificates-bundle
+    - openjdk-17-jre
+    - tomcat
+    - tomcat-native
+    - busybox
+
+work-dir: /usr/share/tomcat/webapps
+
+environment:
+  JAVA_HOME: /usr/lib/jvm/java-17-openjdk
+  CATALINA_HOME: /usr/share/tomcat
+  PATH: /usr/sbin:/sbin:/usr/bin:/bin:/usr/share/tomcat/bin
+
+accounts:
+  groups:
+    - groupname: nonroot
+      gid: 65532
+  users:
+    - username: nonroot
+      uid: 65532
+      gid: 65532
+  run-as: nonroot
+
+paths:
```

Comment:
Should `/usr/share/tomcat` just be owned by the nonroot user?

Have been playing whack-a-mole with these dirs and there could be some that haven't been touched in my testing path.